### PR TITLE
Fix the issue of database write prohibition failure in some versions of postgresql

### PR DESCRIPTION
**Pull request by hanbingleixue**
**What type of PR is this?**

Bug

**What this PR does / why we need it?**

database write prohibition failure in some versions of postgresql

**Which issue(s) this PR fixes？**

Fixes the issue of database write prohibition failure in some versions of postgresql

**Does this PR introduce a user-facing change?**

No

## Checklist
- [x] Make sure there is a [GitHub_issue](https://github.com/huaweicloud/Sermant/issues) related with this PR before you start working on it.
- [x] Make sure you have squashed your change to one single commit.
- [x] GitHub Actions works fine in this PR.

### CODE REVIEW

File: sermant-plugins/sermant-database-write-prohibition/postgresql-42.x-plugin/src/main/java/com/huaweicloud/sermant/postgresqlv42/interceptors/PgStatementInterceptor.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        this.handler = handler;
+    }
+
+    @Override
+    public ExecuteContext doBefore(ExecuteContext context) {
+        PgStatement statement = (PgStatement) context.getObject();
+
+        // Store link information to ensure that QueryExecutorImplInterceptor can retrieve database information
+        try {
+            ThreadDatabaseUrlUtil.setDatabaseUrl(statement.getConnection().getMetaData().getURL());
```
Comment (lilai23) [inline sermant-plugins/sermant-database-write-prohibition/postgresql-42.x-plugin/src/main/java/com/huaweicloud/sermant/postgresqlv42/interceptors/PgStatementInterceptor.java]: consider NPE

Comment (hanbingleixue) [reply sermant-plugins/sermant-database-write-prohibition/postgresql-42.x-plugin/src/main/java/com/huaweicloud/sermant/postgresqlv42/interceptors/PgStatementInterceptor.java]: resolved

**Review by lilai23 — DISMISSED** [review]

File: sermant-plugins/sermant-database-write-prohibition/postgresql-42.x-plugin/src/main/java/com/huaweicloud/sermant/postgresqlv42/utils/QueryExecutorImplEnhancementHelper.java
Comment (daizhenyu) [inline sermant-plugins/sermant-database-write-prohibition/postgresql-42.x-plugin/src/main/java/com/huaweicloud/sermant/postgresqlv42/utils/QueryExecutorImplEnhancementHelper.java:1]: update the class name properly

Comment (hanbingleixue) [reply sermant-plugins/sermant-database-write-prohibition/postgresql-42.x-plugin/src/main/java/com/huaweicloud/sermant/postgresqlv42/utils/QueryExecutorImplEnhancementHelper.java:1]: resolved

**Review by daizhenyu — APPROVED** [review]

**Review by lilai23 — APPROVED** [review]

**Pull request merged**